http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcBaseTest.java

```diff
@@ -215,4 +226,25 @@ protected String getCookieString(HttpServerCookie cookie) {
         return header.toString();
     }
 
+    protected void checkForScopeClaims(Callback callback) {
+        Decoder decoder = java.util.Base64.getUrlDecoder();
```

Comment:
Instead of decoding directly, I think we can use something like this:

`JwtClaims jwtClaims = new JwtConsumerBuilder().setSkipSignatureVerification().setSkipAllValidators().build().processToClaims(token);`

Reply:
Thank you for this! 
I was trying to use jwtClaims, but didn't know how to convert the token to jwt claim. This makes the code a lot cleaner and more efficient now.